news/_posts/2020-12-09-pr-testing-on-prestashop.md

```diff
@@ -0,0 +1,179 @@
+Of course, you’re welcome to fix issues submitted by another contributor (as long as the issue is linked to the PR): this is the magic of open source!
+- The “How to test” field must be filled. It must contain the steps needed to verify the bug is indeed fixed. If possible, it should also contain some information about the possible side-effects of the PR, to help check if there are regressions. Without a clear “how to test”, it’s impossible for us to know what to check on the PR.
+The most important thing about this part is that it must be a functional “How to test”:  PRs are tested by functional QA analysts, not by developers. If your PR requires making technical changes in order to be tested (use a certain hook in the code for example), either attach a test module allowing us to test it, or your PR will need to be tested by the maintainers.
+
+*An example of a clear "How to Test" in a Pull Request:*
+![Example of a clear "How to Test" in a PR](/assets/images/2020/12/clear_howtotest.png)
+
+Other important things:
+
+- The Pull Request needs to have been reviewed and approved by at least two maintainers (for a PR on the Core, only one approval for a PR on a module)
+- All [automated checks](https://build.prestashop.com/news/the-review-process/#automatic-tasks-when-you-open-a-pull-request) must have passed
+- The PR targets the right branch
+
+If there is some information missing, the author, the maintainers or the product team is pinged and the QA analyst waits for this information to start testing the PR.
+
+## Reproduce the bug
+
+The first step of PR testing is reproducing the original bug (on the branch, without the PR): this allows the QA analyst to make sure that the “How to test” is clear and also helps to see what are the possible regressions introduced by the PR.
+
+This is why a complete issue is necessary: the more information it contains, the easier it is to reproduce and test it.
+
+## Check if the Pull Request fixes the bug in the issue
+
+This is the most logical part of PR Testing: after reproducing the bug on the branch, the QA analyst checks out the PR locally and verifies if the bug is indeed fixed, following the instructions in “How to test”.
+
+If the feature is complex enough, or if the “How to test” is not exhaustive, the QA analyst will also test some edge cases in order to make sure that the PR fixes the issue completely.
+
+## Regression testing
+
+This is the most important part of our work: usually, the nominal test is OK, and has already been tested by the author.
+But the PR might have introduced some regressions somewhere else in PrestaShop. This is where our functional knowledge of the solution is really important.
```

Comment:
```suggestion

**But the PR might have introduced some regressions somewhere else in PrestaShop.** This is unfortunately a common issue in software development: modifying a part of the code might introduce side-effects to other parts that are related, directly or not, to the modified part. **This is where our functional knowledge of the solution is really important.**
```
Markdown needs an empty lines between 2 statements to skip a line (see how it's displayed currently here https://github.com/PrestaShop/prestashop.github.io/blob/6b3c66c380fef9c1ce0d5f7177466f7f6a2795d1/news/_posts/2020-12-09-pr-testing-on-prestashop.md)